### DISCUSSION TITLE
Got 403 forbidden on proxy server. Maybe it from open ai or your proxy server？

### DISCUSSION THREAD

**Discussion by puzige** [acheong08/ChatGPT#1152] [Q&A]
```
Client error '403 Forbidden' for url 'https://bypass.duti.tech/api/conversation'
```

**Comment by puzige** [comment]
@acheong08  have a look thx.

**Comment by 0xCoDSnet** [comment]
Also faced with such an error:
`
httpx.HTTPStatusError: Client error '403 Forbidden' for url 'https://bypass.duti.tech/api/conversations?offset=0&limit=20'
For more information check: https://httpstatuses.com/403
`
